VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -383,5 +403,59 @@ await Clients.Group(judge.Username.ToLowerInvariant())
                 _logger.LogError(ex, "Error occured when updating participant device status");
             }
         }
+
+        private async Task<string> GetParticipantUsernameByIdAsync(Guid conferenceId, string participantId)
+        {
+          var username = string.Empty;
+            try
+            {
+                var participantGuidId = Guid.Parse(participantId);
+                var conference = await GetConference(conferenceId);
+                var participant = conference.Participants.SingleOrDefault(x =>
+                    x.Id == participantGuidId);
+
+                if (participant == null)
+                {
+                    _logger.LogDebug($"Participant {participantId} does not exist in conversation");
```

Comment:
LogError?

Reply:
using string interpolation for log messages doesnt add them to the custom dimensions :)

Reply:
done